data/tlg0064/tlg001/tlg0064.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="lat">Periplus Hannonis</title>
+                <author>Hanno</author>
+                <editor xml:lang="lat">CAROLUS MULLERUS</editor>
+                <sponsor>HARVARD COLLEGE LIBRARY</sponsor>
+                <funder/>
+                <principal>Gregory Crane</principal>
+                <meeting/>
+                <respStmt>
+                    <persName xml:id="DDD">Digital Divide Data</persName>
+                    <resp>Corrected and encoded the text</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>RHEA LESAGE</persName>
+                    <resp>Librarian for Hellenic Studies and Coordinator for the Classics Collection
+                        Development</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg0064.tlg001.opp-grc1.xml</idno>
+                <availability>
+                    <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</p>
+                </availability>
+                <date>2016</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United State</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+
+                            <title>Geographi Graeci Minores</title>
```

Comment:
Add xml:lang="lat" to this element. Since the xml:lang was declared as "eng" for the teiHeader, anything that is not English should have the language explicitly marked: "lat", "grc", "deu" etc.